examples/cpp/06_shared_memory/reduction.okl

```diff
@@ -2,17 +2,15 @@
                                        const float *vec,
                                        float *blockSum) {
   // Partial reduction of vector using loop tiles of size block (power of 2)
-  for (int group = 0; group < ((entries + block - 1) / block); ++group; @outer) {
+  for (int group = 0; group < ((entries + block - 1) / block); ++group; @tile(16,@outer)) {
```

Comment:
Any reason this file changed?

Reply:
I am not 100% sure why this was changed. It can be removed.